_includes/header.html

```diff
@@ -18,6 +18,7 @@
             <ul class="dropdown-menu dropdown-menu-end">
               <li><a href="{% link index.html %}#get-started">Get started</a></li>
               <li><a href="{% link index.html %}#use-cases">Use Cases</a></li>
+              <li><a href="https://www.hackster.io/riot-os" target="_blank">Tutorials@Hackster.io<i class="bi bi-box-arrow-up-right align-text-bottom"></i></a></li>
               <li><a href="{% link index.html %}#faq">FAQ</a></li>
```

Comment:
```suggestion
              <li><a href="{% link index.html %}#faq">FAQ</a></li>
              <li><a href="https://www.hackster.io/riot-os" target="_blank">Tutorials@Hackster.io<i class="bi bi-box-arrow-up-right align-text-bottom"></i></a></li>
```

This way the links leading from the riot-os.org page are grouped together.

Reply:
I think the grouping should follow content and not where the content is located. I agree that it looks less nice because the icon indicating external content is now not grouped. Anyhow, in this case, I prefer that design is less prioritized.